ansible/roles/nginx/files/genssl.sh

```diff
@@ -8,22 +8,71 @@ if [ "$#" -ne 1 ]; then
     echo "usage: $0 [common name: host or ip]"
 fi
 CN=$1
+TYPE=$2
+PASSWORD="openwhisk"
 
 ## generates a (self-signed) certificate
 
 ## uncomment to regenerate the key
-#openssl genrsa -out "$SCRIPTDIR/openwhisk-key.pem" 2048
+#openssl genrsa -out "$SCRIPTDIR/openwhisk-server-key.pem" 2048
 
-echo generating certificate request
-openssl req -new \
-    -key "$SCRIPTDIR/openwhisk-key.pem" \
-    -nodes \
+if [ "$TYPE" == "server" ]; then
+    echo generating server certificate request
+    openssl req -new \
+        -key "$SCRIPTDIR/openwhisk-server-key.pem" \
```

Comment:
Since the certificate and key name are configurable in `{{ nginx.ssl.cert }}` parameters, would it make sense to pass these as in input parameter to the script as well?